src/thing.ts

```diff
@@ -247,7 +247,10 @@ export function createThing(options: CreateThingOptions = {}): Thing {
     return thing;
   }
   const name = (options as CreateThingLocalOptions).name ?? generateName();
-  const thing: ThingLocal = Object.assign(dataset(), { name: name });
+  const localSubject: LocalNode = getLocalNode(name);
+  const thing: ThingLocal = Object.assign(dataset(), {
+    localSubject: localSubject,
```

Comment:
For consistency, I've been using a destructuring notation (I think that's the name) to do similar things, e.g. here I would have written 
```typescript
const thing: ThingLocal = Object.assign(dataset(), { localSubject });
```
I imagine this falls under `https://eslint.org/docs/rules/prefer-destructuring`, and if so should we enforce this rule in our coding guidelines, one way or the other ?

Reply:
Yeah I saw that; I've been _not_ doing destructing so far ;) I refrained from pointing that out because if we believe it's important to be consistent in that, it should be a lint rule. And I think that Jack's ESLint config actually contains a rule for that (IIRC it does it your way, though I'm not sure), but we haven't adopted that yet due to a couple of issues (logged in the config's repo, but that's still set to private). So this (or the other instances) will be fixed in due time.